tools/taskcluster/build_targets.py

```diff
@@ -0,0 +1,88 @@
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+This script configures what APKs are built using the options passed in
+from the taskcluster {{ event.version }}.
+
+{{ event.version }} should be in the form: <tag name>:[all,release,debug]-<platform name>:[arm,arm64,x86]
+Some examples of {{ event.version }} and the resultant output from this script
+
+This is the default behaviour with no options. Only the Release build of each
+architecture for each supported platform is built:
+$ python build_targets.py 1.1.4a
+assembleWavevrArmRelease assembleNoapiArmRelease assembleNoapiArm64Release assembleNoapiX86Release assembleSvrArmRelease assembleSvrArm64Release assembleOculusvrArmRelease assembleOculusvrArm64Release assembleGooglevrArmRelease assembleGooglevrArm64Release
+
+Specifies only build the Arm64 version of the OculusVR platform:
+$ python build_targets.py 1.1.4b-oculusvr:arm64
+assembleOculusvrArm64Release
+
+Specifies all build types including Release and Debug:
+$ python build_targets.py 1.1.4c:all
+assembleWavevrArm assembleNoapiArm assembleNoapiArm64 assembleNoapiX86 assembleSvrArm assembleSvrArm64 assembleOculusvrArm assembleOculusvrArm64 assembleGooglevrArm assembleGooglevrArm64
+
+Specifies Release build of Arm64 OculusVR, Arm WaveVR, and x86 NoAPI
+$ python build_targets.py 1.1.4d-oculusvr:arm64-wavevr:arm-noapi:x86
+assembleOculusvrArm64Release assembleWavevrArmRelease assembleNoapiX86Release
+
+Specifies Release and Debug builds of Arm64 OculusVR, Arm WaveVR, and x86 NoAPI:
+$ python build_targets.py 1.1.4e:all-oculusvr:arm64-wavevr:arm-noapi:x86
+assembleOculusvrArm64 assembleWavevrArm assembleNoapiX86
+"""
+import subprocess
+import sys
+
+platforms = {
+   'oculusvr': ['arm', 'arm64'],
+   'wavevr': ['arm'],
+   'googlevr': ['arm', 'arm64'],
+   'noapi': ['arm', 'arm64', 'x86'],
+   'svr': ['arm', 'arm64'],
+}
+
+def findMode(arg):
+   values = arg.split(':')
+   if len(values) > 1:
+     mode = values[1].lower()
+     if mode == 'r' or mode == 'release':
+        return 'Release'
+     elif mode == 'd' or mode == 'debug':
+        return 'Debug'
+     elif mode == 'a' or mode == 'all':
+        return ''
+   return 'Release'
+
+def findArch(mode):
+   archList = platforms.get(mode[0], ['arm'])
+   if len(mode) > 1:
+      value = mode[1]
+      if value == 'a' or value == 'all':
+         return archList
+      elif value.lower() in archList:
+         return [value.lower()]
+   return [archList[0]]
+
+def main(name, argv):
+   targets = ['tag']
+   if len(argv) > 0:
+      targets = argv[0].split('-')
+   mode = findMode(targets[0])
+   size = len(targets)
+   if size == 1:
+      for value in platforms.keys():
+         targets.append(value + ':all')
+   command = []
+   for item in targets[1:]:
+      itemList = item.split(':')
+      if itemList[0] not in platforms:
+         print >> sys.stderr, 'Error: "%s" is not a supported platform' % itemList[0]
```

Comment:
in my testing, this can cause the error message to be included the [shell call](https://github.com/MozillaReality/FirefoxReality/pull/1132/files#diff-ac0229d1171c0983b27003af4c7441a5R129)

Reply:
It didn't when I tested it. Let me try again.

Reply:
nvm, works as expected:

```
echo "["`python tools/taskcluster/build_targets.py 1.1.4e:all-oculusvr:arm64-wavevr:arm-noapi:x86-foobar`"]"
```